announcement/announcement-api/api/src/bundle/announcement.properties

```diff
@@ -336,7 +336,7 @@ perm-annc.all.groups=Access all group announcements
 
 ## Dashboard widget i18n
 #title=Announcements
-viewing=(viewing announcements from the last 10 days)
+viewing=(viewing 3 announcements from the last 10 days)
```

Comment:
this is configurable so you can't hardcode it like this

Reply:
I know that the announcement synoptic widget is configurable, but I don't see a way in the UI for the "dashboard" widget to configure the number of announcements or days.  Would you have to copy all the logic from the synoptic widget to the dashboard widget?